pkg/manager/impl/validation/task_validator.go

```diff
@@ -158,6 +162,11 @@ func requestedResourcesToQuantity(
 				return nil, errors.NewFlyteAdminErrorf(codes.InvalidArgument,
 					"gpu for [%+v] must be a whole number, got: %s instead", identifier, limitEntry.Value)
 			}
+		case core.Resources_EPHEMERAL_STORAGE:
+			err := addResourceEntryToMap(identifier, limitEntry, &requestedToQuantity)
```

Comment:
Does the requestedToQuantity need to be passed as a pointer here. Do we need the function to modify the value

Reply:
yeah it's adding the entry to the map arg